Assets/StreamingAssets/Settings/Settings.xml

```diff
@@ -1,7 +1,7 @@
 <Settings>
   <worldWidth>101</worldWidth>
   <worldHeight>101</worldHeight>
-  <localization>en_US</localization>
+  <localization>modLocalizationExample</localization>
```

Comment:
This wasn't the kind of example I was thinking of. I meant add a localization file to the mods directory for a made up language or something so that we can see how it's loaded, from where and be able to test that it works.